geode-core/src/main/java/org/apache/geode/internal/cache/AbstractRegionMap.java

```diff
@@ -2113,23 +2113,31 @@ public void txApplyPut(Operation p_putOp, Object key, Object nv, boolean didDest
       // "fix" for bug 32440
       Assert.assertTrue(false, "The owner for RegionMap " + this + " is null");
     }
-
-    Operation putOp = p_putOp;
-
-    Object newValue = nv;
-
     final boolean hasRemoteOrigin = !((TXId) txId).getMemberId().equals(owner.getMyId());
     final boolean isTXHost = txEntryState != null;
     final boolean isClientTXOriginator = owner.getCache().isClient() && !hasRemoteOrigin;
     final boolean isRegionReady = owner.isInitialized();
+    boolean onlyExisting = false;
+    if (hasRemoteOrigin && !isTXHost && !isClientTXOriginator) {
+      // If we are not a mirror then only apply the update to existing
```

Comment:
We should probably use "replicate" instead of "mirror" to make it easier for new folks.

Reply:
done